discord/enums.py

```diff
@@ -341,6 +353,9 @@ def category(self):
             AuditLogAction.emoji_update:       AuditLogActionCategory.update,
             AuditLogAction.emoji_delete:       AuditLogActionCategory.delete,
             AuditLogAction.message_delete:     AuditLogActionCategory.delete,
+            AuditLogAction.integration_create: AuditLogActionCategory.create,
+            AuditLogAction.integration_update: AuditLogActionCategoty.update,
```

Comment:
You have a mistype here,
AuditLogActionCategoty
Instead of
AuditLogActionCategory

Reply:
This was already fixed.

Reply:
See https://github.com/Rapptz/discord.py/commit/f9b8b234e314cd182c39ec0a55f466d81b9926c7.